README.md

```diff
@@ -11,15 +11,31 @@ Tested with Python 3.11.4
 The following steps are not required but recommended. This will allow you to install packages in your isolated virtual environment instead of globally, reducing the risk of breaking system tools or other projects.
 
 1. Install [pyenv](https://github.com/pyenv/pyenv) and the [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv) plugin.
-2. Download the appropriate Python version with `pyenv install 3.11.4` in the command line.
-3. Create a virtual environment with the appropriate Python version and name for your environment, for example `pyenv virtualenv 3.11.4 innovationserhebung`
-4. Activate the environment with `pyenv activate innovationserhebung`
+2. Download the appropriate Python version with 
+```shell
+pyenv install 3.11.4
+``` 
+in the command line.
+3. Create a virtual environment with the appropriate Python version and name for your environment, for example 
+```shell
+pyenv virtualenv 3.11.4 innovationserhebung
+```
+4. Activate the environment with 
+```shell
+pyenv activate innovationserhebung
+```
 
 ### Run the APP
 
 1. Move to the `/app` directory.
-2. Install the required libraries with the command line `pip install -r requirements.txt`
-3. Run the server with the command line `uvicorn main:app --reload`
+2. Install the required libraries with the command line 
+```shell
+pip install -r requirements.txt
+```
+3. Run the server with the command line 
+```shell
+uvicorn main:app --reload
```

Comment:
Installing `uvicorn` could be mentioned further up in the "Set your virtual environment" section, because not everyone might have uvicorn installed?

The same goes for the `tailwindcss` CLI tool (although you could put this into `devDependencies` and then add a simple script in `package.json`, so devs don’t need to install tailwindcss globally 💡).

Reply:
Ah, very good.